### DISCUSSION TITLE
Welcome to wax Discussions!

### DISCUSSION THREAD

**Discussion by jeffbarish** [jeffbarish/wax#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jrayner00** [comment]
Jeff, thank you for your continued work on Wax - I am absolutely loving it! Looking forward to updating to v1.2.0.

**Reply by jeffbarish** [reply]
I am absolutely delighted! The new feature in v1.2.0 is a small improvement, but it can be helpful when your collection gets large. By the way, you could help make Wax better by occasionally checking the wax.log file in ~/wax/.config/log. If you ever find a traceback in there, post it so that I can fix the problem. Also, feel free to request new features, although I do not guarantee that I will implement them.
